src/components/ListItem.jsx

```diff
@@ -21,6 +21,14 @@ export function ListItem({ name, data, listToken }) {
 		}
 	};
 
+	function handleDeleteItem() {
+		if (window.confirm(`Are you sure you wish to delete ${name}?`)) {
+			deleteItem(listToken, data.id);
+		} else {
+			this.onCancel();
```

Comment:
@louieknolle @tinazarb 
Seeing console errors when clicking `Cancel` button

Reply:
I am not familiar with this bit of a syntax: `this.onCancel()`. As far as I know, you don't need to cancel a button onClick or window.confirm like this. Let me know if you read about this somewhere, I'd love to see where this idea came from. But I think you can remove the whole `else` part of this if statement.